concepts/translations/fallback-language-selection.md

```diff
@@ -24,16 +24,40 @@ The table shows some examples of common cases:
 | **`fr`**      | `fr-FR` (French in France)          | `fr-CA`, `fr-CH`          |
 | **`nl`**      | `nl-NL` (Dutch in the Netherlands)  | `nl-BE`                   |
 
+## Migration and linting via command
+
+To support these processes, the `LintTranslationFilesCommand` can be executed using `bin/console translation:lint-filenames` to validate translation filenames.
+
+The command outputs tables for each domain (Administration, Core/base files, and Storefront) containing the following information:
+
+- **Filename** – The name of the translation file, e.g. `storefront.de.json`.
+- **Path** – The file path where this translation file was found, e.g. `src/Storefront/Resources/snippet`.
+- **Domain** – The prefix of the corresponding storefront translation file. For administration files, `administration` is shown in this column. For extensions, it can be helpful to name storefront files accordingly, for example `cms-extensions.en.json`. Please note: language-defining base files **must always** use `messages` here!
+- **Locale** – The language code following [IETF BCP 47](https://datatracker.ietf.org/doc/html/bcp47), restricted to [ISO 639-1 (2-letter) language codes](https://en.wikipedia.org/wiki/ISO_639-1). Example: `de-DE` for German (Germany).
+- **Language** – The first part of the locale, representing the language used. Example: `de` (German) when the full locale is `de-DE`.
+- **Script** – Specifies the writing system used for the language when multiple scripts exist. This part is optional and rarely used, as most Shopware processes currently do not support or distinguish between scripts. For example, Serbian (Serbia) can be written in both Cyrillic and Latin (`sr-Cyrl-RS` vs. `sr-Latn-RS`).
```

Comment:
We could mention that the script is not included in the standard Shopware implementation and exists only for extensibility purposes

Reply:
Changed to `This part is optional and rarely used, as Shopware processes currently do not support or distinguish between scripts and only offer it for extensibility. `